DigitalLearningSolutions.Data/Helpers/DateHelper.cs

```diff
@@ -3,10 +3,81 @@
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Data.Models.TrackingSystem;
 
     public static class DateHelper

```

Comment:
Namespace for this file is "DigitalLearningSolutions.Web.Helpers" despite it being in "DigitalLearningSolutions.Data.Helpers"